src/lib/containers/IDUReportSubmissionInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Skeleton } from '@material-ui/lab'
+import { SubmissionInfo } from '../utils/synapseTypes/SubmissionInfo'
+import { SMALL_USER_CARD } from '../utils/SynapseConstants'
+import UserCard from './UserCard'
+import { formatDate } from '../utils/functions/DateFormatter'
+import moment from 'moment'
+import { SkeletonTable } from '../assets/skeletons/SkeletonTable'
+
+export type SubmissionInfoCardProps = {
+  info: SubmissionInfo
+}
+
+export const SubmissionInfoCard: React.FunctionComponent<
+  SubmissionInfoCardProps
+> = ({ info }: SubmissionInfoCardProps) => {
+  return (
+    <>
+      {info && (
+        <p className="SubmissionInfoCard">
+          <strong> Project Lead: </strong> <span>{info.projectLead}</span>{' '}
```

Comment:
Would it be a big change to use the Typography styled components?

Reply:
Initially matching SWC implementation.  There are old IDUs that are hard-coded in the wiki, and this matches their styling.